pkg/policyreport/reportrequest.go

```diff
@@ -229,10 +187,20 @@ func (gen *Generator) handleErr(err error, key interface{}) {
 	logger.Error(err, "failed to process report request", "key", key)
 	gen.queue.Forget(key)
 	gen.dataStore.delete(keyHash)
-	gen.changeRequestMapper.decrease(keyHash)
 }
 
-func (gen *Generator) processNextWorkItem() bool {
+func (gen *generator) getReportChangeRequestCount(ns string) int {
+	if ns == "" {
+		items, _ := gen.clusterReportChangeRequestLister.List(labels.Everything())
```

Comment:
Does this get all RCRs or RCRs for the cluster-wide policy report i.e. one without a namespace? 

We don't want to process all RCRs here, right?

Reply:
All RCRs are created in the kyverno namespace, we use a label to target a specific namespace.

Reply:
We don't process any RCR here, just counting to compare with the configured limit.

Reply:
@JimBugwadia if i'm not mistaken cluster-wide policy reports use CRCRs (not RCRs) right ?

Reply:
You are right - there is a CRCR!

In that case do we need to handle the case for an empty namespace ("") and treat it as Default or can we simply remove the  if block?